spec.bs

```diff
@@ -982,20 +991,40 @@ navigables</a> section, add the following:
   #### Saved queries #### {#saved-queries}
 
   <div algorithm>
```

Comment:
Suggest adding a brief note below this algorithm, describing what the return value means. (-2, -1, >=0)

Reply:
Done.